frontend/app/utils/auth.js

```diff
@@ -1,49 +1,58 @@
-import { setCookie, destroyCookie } from 'nookies';
-export const handleSignUp = async (name, email, password, phone, role) => {
+export const handleSignUp = async data => {
   try {
-    const response = await fetch('/api/users', {
+    const response = await fetch(`/api/users`, {
       method: 'POST',
       headers: { 'Content-Type': 'application/json' },
-      body: JSON.stringify({ name, email, password, phone, role }),
+      body: JSON.stringify(data),
     });
 
-    const data = await response.json();
+    const result = await response.json();
+    console.log(result);
+
     if (response.ok) {
-      return { success: true, role: data.role };
+      alert(result.message + ' Sign In again.');
```

Comment:
here, on the other hand, is a great usecase to use string literal. It's generally better practice than concatenating with `+` :)